pkg/configmap/configmap.go

```diff
@@ -169,10 +169,16 @@ func (s *Sync) add(obj interface{}) {
 	}
 }
 
-func (s *Sync) update(_, obj interface{}) {
+func (s *Sync) update(oldObj, obj interface{}) {
 	cm := obj.(*v1.ConfigMap)
 	if match, isPolicy := s.matcher(cm); match {
 		s.syncAdd(cm, isPolicy)
+	} else {
+		// check if the label was removed
+		oldCm := oldObj.(*v1.ConfigMap)
```

Comment:
I thought that it could not be called with a different type because of the way  `Informer` and `ListWatch` were created (only for the config maps): [Informer](https://github.com/open-policy-agent/kube-mgmt/blob/master/pkg/configmap/configmap.go#L147), [ListWatch](https://github.com/open-policy-agent/kube-mgmt/blob/master/pkg/configmap/configmap.go#L142).
Also, all the other code doesn't make the type assertion check.

Maybe I miss some case here?

Reply:
Ah, that's fair. I thought we had defensive checks elsewhere but you're right.